sourcegraph/sourcegraph
# [fixes #2761] Update remote selection to prefer 'upstream' and 'origin'

## Test plan

- Add a new remote to the git repo `git remote add`
- Observe that the remotes list now returns two results and that `origin` is preferred
